CHANGES.rst

```diff
@@ -9,6 +9,11 @@ Features
 
 - Add ``only_rerun`` and ``rerun_except`` arguments to ``@pytest.mark.flaky`` marker.
 
+Bug fixes
++++++++++
+
+-- Fix handling of ``--rerun-except`` and ``--only-rerun`` when used together,
+   and multiple instances of ``--rerun-except``.
```

Comment:
This bugfix changes the behaviour, so could you please write about the behaviour before and after the fix here as you did in the PR description?